docs/vocabulary/licenses/20210623.html

```diff
@@ -95,8 +95,8 @@ <h2>List of licenses for Swiss opendata to enter in dct:license</h2>
                   >
                 </td>
                 <td>
-                  Terms of Use opendata.swiss
-                  "NonCommercialAllowed-CommercialAllowed-ReferenceNotRequired"
+                  <b>Open use</b>
+                <br>Terms of Use opendata.swiss</br> 
```

Comment:
In general, it's better not to put text inside a `<br>` element like this, for accessibility (see https://developer.mozilla.org/en-US/docs/Web/HTML/Element/br#accessibility_concerns). I would suggest just using a single self-closing br tag, like this:

```suggestion
                  <b>Open use</b><br />
                  Terms of Use opendata.swiss
```

(and the same for the other three terms of use, of course).